concepts/05 UI Components/TreeView/00 Getting Started with TreeView/70 Enhance Performance on Large Datasets.md

```diff
@@ -0,0 +1,7 @@
+If the TreeView is bound to a large dataset, the component can load data and render nodes on demand as a user expands them. Because this tutorial uses only a small dataset, this functionality is not demonstrated here. You can refer to the following demos for information on how to configure it: [Virtual Mode](https://js.devexpress.com/Demos/WidgetsGallery/Demo/TreeView/VirtualMode/) and [Load Data on Demand](/Demos/WidgetsGallery/Demo/TreeView/LoadDataOnDemand).
```

Comment:
I guess Dirk would have flipped this sentence:

>  Because this tutorial uses only a small dataset, this functionality is not demonstrated here.

And I tend to share his views. This would sound more natural.

```suggestion
If the TreeView is bound to a large dataset, the component can load data and render nodes on demand as a user expands them. This functionality is not demonstrated in this tutorial because it uses only a small dataset. You can refer to the following demos for information on how to load data on demand: [Virtual Mode](https://js.devexpress.com/Demos/WidgetsGallery/Demo/TreeView/VirtualMode/) and [Load Data on Demand](/Demos/WidgetsGallery/Demo/TreeView/LoadDataOnDemand).
```